pkg/controller/reconciler_test.go

```diff
@@ -0,0 +1,175 @@
+package controller
+
+import (
+	"testing"
+	"time"
+
+	"github.com/Bonial-International-GmbH/ingress-monitor-controller/pkg/config"
+	"github.com/Bonial-International-GmbH/ingress-monitor-controller/pkg/monitor/fake"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	fakeclient "sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func TestIngressReconciler_Reconcile(t *testing.T) {
```

Comment:
Name does not meet naming conventions. Suggestion: TestIngressReconcilerReconcile